### DISCUSSION TITLE
Add read more processing for Module and Full Display

### DISCUSSION THREAD

**Discussion by bcordis** [Joomla-Bible-Study/Proclaim#936] [Ideas]
`Add an auto read more button similar to other modules. This would give a better look and feel to the display of code. We would also need to have a strip code of read more from displaying on the complete article.

**Comment by tomfuller2** [comment]
This sounds good. I assume we do it as Joomla does?

**Comment by bcordis** [comment]
Yes, this would be best to do that like Joomla.
